automatic/nexus-repository/nexus-repository.nuspec

```diff
@@ -40,6 +40,12 @@ You can pass parameters as follows:
 
 ## Notes
 
+- **ATTENTION BREAKING CHANGE FOR UPGRADES FROM VERSIONS BEFORE 3.71.0.6**  
+    Nexus has upgraded the bundled version of Java, and sunset the OrientDb database type.
+    This means that you will need to migrate your database from OrientDb, if you are using it.
+    The version after 3.71.0.6 will attempt to migrate the database during the package upgrade.
```

Comment:
after 3.71.0.6? 

This reads like this version won't migrate to H2, when it in fact wil.

Reply:
To be clear, version 3.71.0.6 will never have the upgrade logic in. :(

Reply:
Ooooh right, as that's already hit the feed approved. Derp. Ignore me.